python/ql/test/library-tests/ApiGraphs/py3/test_captured_flask.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_user import UserMixin
+
+def create_app():
+  app = Flask(__name__)
+  db = SQLAlchemy(app)  #$ use=moduleImport("flask_sqlalchemy").getMember("SQLAlchemy").getReturn()
+
+  class Users(db.Model, UserMixin):  #$ use=moduleImport("flask_sqlalchemy").getMember("SQLAlchemy").getReturn().getMember("Model").getASubclass()
+      __tablename__ = 'users'
+
+  @app.route('/v2/user/<int:id>', methods=['GET','PUT'])
+  def users(id):
+      if 'Authorization-Token' not in request.headers:
+          return make_response(jsonify({'Error':'Authorization-Token header is not set'}),403)
+
+      token = request.headers.get('Authorization-Token')
+      sid = check_token(token)
+
+      #if we don't have a valid session send 403
+      if not sid:
+          return make_response(jsonify({'Error':'Token check failed: {0}'.format(sid)}))
+      try:
+          user = Users.query.filter_by(id=id).first()  #$ use=moduleImport("flask_sqlalchemy").getMember("SQLAlchemy").getReturn().getMember("Model").getASubclass().getMember("query").getMember("filter_by")
+      except Exception as e:
+          return make_response(jsonify({'error':str(e)}),500)
```

Comment:
We don't actually need the rest of the rest of the error handling for the example to be valid, do we?

```suggestion
  @app.route('/v2/user/<int:id>', methods=['GET','PUT'])
  def users(id):
      Users.query.filter_by(id=id).first()  #$ use=moduleImport("flask_sqlalchemy").getMember("SQLAlchemy").getReturn().getMember("Model").getASubclass().getMember("query").getMember("filter_by")
```